Description: Fix some spelling mistakes in upstream's strings.
Forwarded: accepted by upstream in PR #119
Author: Sebastian Kuzminsky <seb@highlab.com>
Index: yosys/backends/json/json.cc
===================================================================
--- yosys.orig/backends/json/json.cc	2016-02-24 15:41:34.500689004 -0700
+++ yosys/backends/json/json.cc	2016-02-27 18:55:38.342523778 -0700
@@ -411,10 +411,10 @@
 		log("      - the inverted value of the specified input port bit\n");
 		log("\n");
 		log("    [ \"and\", <node-index>, <node-index>, <out-list> ]\n");
-		log("      - the ANDed value of the speciefied nodes\n");
+		log("      - the ANDed value of the specified nodes\n");
 		log("\n");
 		log("    [ \"nand\", <node-index>, <node-index>, <out-list> ]\n");
-		log("      - the inverted ANDed value of the speciefied nodes\n");
+		log("      - the inverted ANDed value of the specified nodes\n");
 		log("\n");
 		log("    [ \"true\", <out-list> ]\n");
 		log("      - the constant value 1\n");
@@ -445,7 +445,7 @@
 		log("    ]\n");
 		log("\n");
 		log("Future version of Yosys might add support for additional fields in the JSON\n");
-		log("format. A program processing this format must ignore all unkown fields.\n");
+		log("format. A program processing this format must ignore all unknown fields.\n");
 		log("\n");
 	}
 	virtual void execute(std::ostream *&f, std::string filename, std::vector<std::string> args, RTLIL::Design *design)
Index: yosys/manual/command-reference-manual.tex
===================================================================
--- yosys.orig/manual/command-reference-manual.tex	2016-02-27 16:24:11.177827043 -0700
+++ yosys/manual/command-reference-manual.tex	2016-02-27 18:55:42.814481159 -0700
@@ -3783,10 +3783,10 @@
       - the inverted value of the specified input port bit
 
     [ "and", <node-index>, <node-index>, <out-list> ]
-      - the ANDed value of the speciefied nodes
+      - the ANDed value of the specified nodes
 
     [ "nand", <node-index>, <node-index>, <out-list> ]
-      - the inverted ANDed value of the speciefied nodes
+      - the inverted ANDed value of the specified nodes
 
     [ "true", <out-list> ]
       - the constant value 1
@@ -3817,7 +3817,7 @@
     ]
 
 Future version of Yosys might add support for additional fields in the JSON
-format. A program processing this format must ignore all unkown fields.
+format. A program processing this format must ignore all unknown fields.
 \end{lstlisting}
 
 \section{write\_smt2 -- write design to SMT-LIBv2 file}
